cms/static/sass/elements/_modal-window.scss

```diff
@@ -760,19 +761,20 @@
     // UI: staff lock and discussion
     .edit-discussion,
     .edit-staff-lock,
+    .edit-optional-content,
     .edit-settings-timed-examination,
     .edit-unit-access {
       .checkbox-cosmetic .input-checkbox {
         @extend %cont-text-sr;
 
         // CASE: unchecked
-        ~ .tip-warning {
+        ~.tip-warning {
           display: block;
         }
 
         // CASE: checked
         &:checked {
-          ~ .tip-warning {
+          ~.tip-warning {
```

Comment:
This does not seem to be right. Why did we change it?

Reply:
sorry, autoformatting strikes again